client/consensus/Cargo.toml

```diff
@@ -47,10 +47,12 @@ cumulus-relay-chain-interface = { workspace = true }
 
 # Polkadot
 polkadot-node-primitives = { workspace = true }
+polkadot-node-subsystem = { workspace = true }
```

Comment:
Where are we using this dependency?

Reply:
It is used for some types inside the lookahead collator https://github.com/moondance-labs/tanssi/blob/agustin-lookahead-collator/client/consensus/src/collators/lookahead.rs#L45

Reply:
thanks